react/features/base/participants/components/ParticipantView.native.js

```diff
@@ -64,6 +73,12 @@ class ParticipantView extends Component {
          */
         avatarStyle: PropTypes.object,
 
+        /**
+         * Indicates if this component is being rendered in the large view or
+         * not.
+         */
+        isLargeView: PropTypes.bool,
```

Comment:
I'm not 100% convinced this is the right way to go. On one hand it seems like a bad abstraction, but on the other, we can't really draw stuff on top of this view in the large view without being aware of its layout, so I went for this. Thoughts?

Reply:
how about renaming it to something that indicates what functionality depends on this flag ?

Reply:
for example "displayInterruptedStatus" or maybe you can come up with something better

Reply:
Sounds like a great idea!